### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by NechiforMaria**
Check the functionality for menu,submenu, hamburger menu, and footer

### CODE REVIEW

File: tests/Home.js [hunk 1]
```diff
@@ -26,108 +24,131 @@ describe("Home page", () => {
...
 
+  it("Check the last part of footer functionality", async () => {
+    await HomePage.scrollDown();
+    await (await HomePage.dmcaFooterButton).waitForClickable();
     await (await HomePage.dmcaFooterButton).click();
+    await (await Menu.title).waitForDisplayed();
     await expect(await Menu.title).toHaveText("DMCA Policy");
     await HomePage.backPage();
 
+    await HomePage.scrollDown();
```
Comment (maria-belciug) [inline tests/Home.js:148]: I don't see this function added on this PR? it is on master already?

File: tests/CategoryAndArticles.js [hunk 2]
```diff
@@ -44,160 +38,154 @@ describe("Category and articles for category", () => {
...
     await expect(await browser).toHaveUrl("https://astage.honesttopaws.com/");
 
     await (await HomePage.omgArticle4).waitForDisplayed();
     await (await HomePage.omgArticle4).click();
-    await Menu.waitForExistHeader();
     await (await AllArticles.galleryCategory).waitForDisplayed();
     await expect(await AllArticles.galleryCategory).toHaveText("Omg");
+  });
+
+  it("Click on all see more button", async () => {
```
Comment (maria-belciug) [inline tests/CategoryAndArticles.js]: This title is not so easy to understand. You specify that you click on all "See more button". There is only one see more button? There is a chrome extension that I think will help you for English writing: [Grammarly for Chrome](https://chrome.google.com/webstore/detail/grammarly-for-chrome/kbfnbcaeplbcioakkpcpgfkobkghlhen)

File: tests/Home.js (hunk 1 above)
Comment (maria-belciug) [reply tests/Home.js:148]: If you  merge this PR and this function is not on the main branch you need to add the file where you wrote this function: scrollDown() on this PR

File: page-objects/pages/AllArticles.js [hunk 3]
```diff
@@ -1,13 +1,16 @@
...
-  //   await titleArticles.forEach(async (titleArticles) => {
+  //   const titleArticles = await $$(".c-link__cover a");
+  //   await titleArticles.forEach(async (titleArticle) => {
   //     {
-  //       await titleArticles.click();
-  //       console.log(titleArticles);
-  //       Homepage.backPage();
+  //       await titleArticle.click();
+  //       console.log(titleArticle);
+  //       await Homepage.backPage();
```
Comment (maria-belciug) [inline page-objects/pages/AllArticles.js:13]: I suppose this file is part of another branch and should not be on this PR

File: tests/Home.js (hunk 1 above)
Comment (NechiforMaria) [reply tests/Home.js:148]: I added the function in this PR

File: tests/CategoryAndArticles.js (hunk 2 above)
Comment (NechiforMaria) [reply tests/CategoryAndArticles.js]: No, there are more buttons. I did not realize

File: page-objects/pages/AllArticles.js (hunk 3 above)
Comment (NechiforMaria) [reply page-objects/pages/AllArticles.js:13]: Yes

**Review by maria-belciug — APPROVED** [review]

**Pull request merged**